BackofficeBundle/Resources/public/coffee/dataTable/DataTableView.coffee

```diff
@@ -0,0 +1,298 @@
+OpenOrchestraDataTableChannel = new (Backbone.Wreqr.EventAggregator)
+
+###*
+* @class DataTableView
+###
+class DataTableView extends OrchestraView
+
+  tagName: 'div'
+
+  events:
+    'init.dt table': 'initComplete'
+    'processing.dt table': 'processingData'
+
+  defaultSettings:
+      serverSide: true
+      processing: true
+      pageLength: 10
+      page: 0
+      searching: true
+      ordering: true
+      orderCellsTop: true
+      autoWidth: false
+      stateSave: true
+      pagingType: 'input_full'
+      globalSearch: false
+      tableClassName: ''
+
+  ###*
+  * required options
+  * {
+  *   columns: {object}
+  *   columnDefs: {object}
+  *   tableId : {string}
+  * }
+  * if serverSide is activate (by default it is enabled),
+  * you should also give url option
+  *
+  * @param {Object} options
+  ###
+  initialize : (options) ->
+    @options = options
+    @api = null
+    @settings = {}
+    OpenOrchestraDataTableChannel.bind 'draw', @draw, @
+
+  ###*
+  * @return {this}
+  ###
+  render: ->
+    settings = $.extend true, {}, @defaultSettings, @options
+
+    $.extend $.fn.dataTableExt.oStdClasses, {
+      "sWrapper": "dataTables_wrapper form-inline",
+      "sFilterInput": "form-control",
+      "sLengthSelect": "form-control"
+    }
+
+    if(not settings.displayStart?)
+      settings.displayStart = settings.pageLength * settings.page
+
+    if(not settings.buttons?)
+       settings.buttons = @getButtonsSettings(settings);
+
+    if(settings.serverSide? and true == settings.serverSide)
+        $.extend true, settings, @getServerSideSettings(settings)
+        OpenOrchestraDataTableChannel.bind 'clearCache', @clearCache, @
+
+    if(settings.stateSave? and true == settings.stateSave)
+        $.extend true, settings, @getStateSaveSettings(settings)
+
+    if(not settings.dom?)
+      settings.dom = @getDomSettings(settings);
+
+    table = $("<table></table>")
+    table.addClass(settings.tableClassName)
+    table.attr('id', 'dt-' + settings.tableId)
+
+    @settings = settings
+    @api = table.DataTable(settings)
+
+    $(this.el).append(table);
+
+    return @
+
+  ###*
+  * @param {object} e jquery event
+  * @param {Object} settings DataTable settings
+  * @param {boolean} processing
+  ###
+  processingData : (e, settings, processing) ->
+    message = settings.oLanguage.sProcessing
+    domProcessing = $("<div class='dataTables_processing'><i class='fa fa-cog fa-spin'></i> "+message+"</div>")
```

Comment:
Is there any solution to remove this html from the js ? Using an underscore template for instance